docs/UserGuide.md

```diff
@@ -217,7 +217,7 @@ Format: `delete company INDEX`
 * Deletes the company at the specified `INDEX`.
 * The index refers to the index number shown in the displayed company list.
 * The index **must be a positive integer** 1, 2, 3, …​
-* Deleting a company with jobs attributed to it also causes those jobs to be deleted.
+* Deleting a company with jobs attributed to it also causes those jobs to be deleted. Any contacts that matched to those jobs will not be deleted, however they will be unmatched with those job.
```

Comment:
Suggest to reword to `Any contacts that **_are_** matched to ... unmatched with those **_jobs_**`

Reply:
Fixed!